app/controllers/pg_hero/home_controller.rb

```diff
@@ -72,11 +72,16 @@ def cluster_info
       @citus_enabled = @database.citus_enabled?
       @citus_version = @database.citus_version
       @nodes_info = @database.nodes_info
+      @partitioned_tables = @database.partitioned_tables
```

Comment:
partitioned table is something different and that name should not be used for distributed tables. Yes we have pg_dist_partition table but it is very, very old name.

Reply:
How about dist_tables_extended ?

Reply:
makes sense